### PR TITLE
docs: finish cli commands, except quota increase, which doesn't have …

### PR DESCRIPTION
**Pull request by fernandoataoldotcom**
…good cli options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,77 @@
...
+
 2. Set env variable while in the directory with the `creds.json`:
 
     ```bash
 
     export GOOGLE_CREDENTIALS=$(pwd)/creds.json
     ```
 
-3. In the project, [grant the service account the `Owner` role](https://console.cloud.google.com/iam-admin/iam) by selecting `GRANT ACCESS` at the top of the screen and filling out the principal (your service account role) and the `Owner` role.
+3. In the project, [grant the service account the `Owner` role](https://console.cloud.google.com/iam-admin/iam) by selecting `GRANT ACCESS` at the top of the screen and filling out the principal (service account role) and the `Owner` role.
```
Comment (venkatamutyala) [inline README.md:61]: Possible to do this with the cli @fernandoataoldotcom ?

Comment (venkatamutyala) [inline README.md:61]: Since you do this below with the CLI I think we should just remove this?

Comment (fernandoataoldotcom) [reply README.md:61]: ya, sounds good

**Review by venkatamutyala — CHANGES_REQUESTED** [review]
remove manual steps since you already mention the CLI command for setting roles/owner permissions

File: README.md (hunk 1 above)
Comment (fernandoataoldotcom) [reply README.md:61]: I'll leave the links in as docs, but remove the instructions

**Comment by venkatamutyala** [comment]
@fernandoataoldotcom I think it's worth moving up the permissions to be before the download. This way they don't switch context from cloud -> local -> cloud.

**Review by venkatamutyala — APPROVED** [review]

**Pull request merged**